lib/models/obms.js

```diff
@@ -106,6 +108,37 @@ function ObmsModelFactory (
             next();
         },
 
+        publishEvent: function(nodeId, next) {
+            return waterline.nodes.findByIdentifier(nodeId)
+            .then(function(node) {
+                if (!_.isEmpty(node)) {
+                    return waterline.obms.find({node: node.id})
```

Comment:
If 'created/updated' is calling this can you skip going back to the database and assert that it is 'managed'.